mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala

```diff
@@ -78,16 +78,19 @@ final class Bucketizer @Since("1.4.0") (@Since("1.4.0") override val uid: String
   def setOutputCol(value: String): this.type = set(outputCol, value)
 
   /**
-   * Param for how to handle invalid entries. Options are 'skip' (filter out rows with
-   * invalid values), 'error' (throw an error), or 'keep' (keep invalid values in a special
-   * additional bucket).
+   * Param for how to handle invalid entries containing either NaN or null values.
```

Comment:
This is actually not true currently - `null` is not handled as an invalid value - see https://issues.apache.org/jira/browse/SPARK-19781